core-bundle/contao/templates/twig/backend/data_container/show.html.twig

```diff
@@ -0,0 +1,23 @@
+{% trans_default_domain "contao_default" %}
+
+{% for table, rows in data %}
+    {% for entries in rows %}
+        <table class="tl_show with-padding with-zebra">
+            <thead>
+                <tr>
+                    <th class="tl_label">{{ 'MSC.table'|trans }}</th>
+                    <th>{{ table }}</th>
+                </tr>
+            </thead>
+
+            <tbody>
+                {% for label, value in entries %}
+                    <tr>
+                        <td class="tl_label">{{ label|raw }}</td>
```

Comment:
A label in the core looks like `foo <small>bar</small>` or `-`. I would have loved to concatenate the HTML in the template and drop the `|raw` but unfortunately the `onshow_callback` receives the readily constructed HTML (and can change it).